apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerOverviews.cs

```diff
@@ -0,0 +1,283 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Threading.Tasks;
+using Microsoft.Practices.EnterpriseLibrary.TransientFaultHandling;
+using Xunit;
+
+#if NET451 || NET451 || NET452
+using System.Transactions;
+#endif
+
+namespace Google.Cloud.Spanner.Data.Snippets
+{
+    [Collection(nameof(SnippetFixture))]
+    public class SpannerOverviews
+    {
+        private readonly SnippetFixture _fixture;
+        private string _projectid;
+        private string _instanceName;
+        private string _databaseName;
+
+        public SpannerOverviews(SnippetFixture fixture)
+        {
+            _fixture = fixture;
+            _projectid = _fixture.TestProjectName;
+            _instanceName = _fixture.TestInstanceName;
+            _databaseName = _fixture.TestDatabaseName;
+        }
+
+        [Fact]
+        public async Task CreateDatabaseAsync()
+        {
+            // CreateDatabaseAsync will create a new database and dispose of it right after.
+            // But for consistency sake, we want to show the same variables in the snippets.
+            _databaseName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
```

Comment:
yeah its ok -- xUnit creates a new instance of SpannerOverviews for each test method, so setting this only affects this test.  As the comment says, I'm only doing this to keep the snippets consistent and I don't want to have locals for each method.